code/modules/spells/targeted/roidrat_punch.dm

```diff
@@ -0,0 +1,78 @@
+/spell/targeted/roidrat_punch // A much less powerful version of the wizard punch. Can launch a mob a long distance, but causes no explosion. Even this rat's gains have their limits
```

Comment:
could this be refactored to a child of wizard punch for better maintainability?

Reply:
I'll work on this later tonight and see if I can get it to work.

Reply:
This has been done. Probably still has a lot of redundancy, but it's a child of the wizard punch spell now.